node/pkg/watchers/stacks/fetch.go

```diff
@@ -0,0 +1,305 @@
+package stacks
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/certusone/wormhole/node/pkg/common"
+)
+
+type (
```

Comment:
Can you link to the JSON documentation for each of these types? it would be helpful to have a link in case the responses change and these types need to be updated.